src/components/Home/Today1Hourly/Today1Hourly.tsx

```diff
@@ -53,6 +54,9 @@ class Today1Hourly extends React.Component<Today1HourlyProps, Today1HourlyState>
 
   render() {
     const { data, loading, error } = this.state;
+    if (window.screen.width < 650) {
```

Comment:
i would recommend to move the condition in parent component. reason: logic of showing/hiding component doesn't belong to component itself and you might have another page where you need to show it but with no restrictions about screen width

Reply:
ok, moved